config/src/main/java/io/quarkus/ts/configmap/api/server/HelloResource.java

```diff
@@ -34,4 +41,17 @@ public HelloConfigProperties getProperties() {
         return configProperties;
     }
 
+    @Path("/secret")
+    @GET
+    @Produces(MediaType.TEXT_PLAIN)
+    public String getSecret() {
+        return secret;
+    }
+
```

Comment:
Please also add there 
```
 @Inject
  SmallRyeConfig config;
```
and endpoint that access the value e.g. `config.getConfigValue("secrets.my.secret")`, I know ATM it's tested upstream, but in my eyes, it's only actual value of this feature. (just my opinion)

Reply:
This config will not have `.withSecretKeys("secret.password")` in its builder and therefore will not have any keys marked as secret. I guess, we can create a separate bean for that, but  for now the idea is to verify, that:
1) We can inject properties and use them as is
2) We can build our own config and mark some properties as secret

Reply:
Ah, good, I wasn't sure if that's what is happening. But that basically means you are not testing primary way user is going to do, right? How many users are going to build config manually? They are going to use injected config, so we need an interceptor and add some secret (f.e. under another name).

Reply:
Btw. that's not very user friendly. what you can't provide secrets without interceptor, e.g. config property that says "secure xyz"

Reply:
Hmm, I wanted to argue, but it looks like[1], that config interceptors are supported in Quarkus. Good idea, will add this to tests.

[1] https://quarkus.io/guides/config-extending-support#config-interceptors

Reply:
Thank you